oauth/authentication-oauth2.0.md

```diff
@@ -88,36 +95,45 @@ GET https://app.solvexia.com/authorize?client_id=DDF-AAFBD447-55432-475B-83DB-B5
 | ------------- |------------- | -------------|
 |client_id|`string`|REQUIRED. Client id that you received when you created an application in SolveXia.|
 |response_type|`string`|REQUIRED. will always be "code".|
-|redirect_uri|`string`|OPTIONAL. If not supplied it will use configured redirect url of the oauth application, if supplied it must match the oauth application redirect url|
-
-##### Response
+|redirect_uri|`string`|OPTIONAL. If not supplied it will use configured redirect url of the oauth application, if supplied it must match the oauth application redirect url.|
+|state|`string`|OPTIONAL. Specifies any string value that your application uses to maintain state between your authorization request and the authorization server's response.|
 
-Will redirect to login page
+After SolveXia authorization server successfully verifies your parameters it will redirect user to the login page and further authorization form.
 
-#### 2. Manually login
+#### 2. SolveXia propmts user to login and consent
 ![login](login.png)
 
-#### 3. Authorize application to act on your behalf
+In this step, the user logs in and considers whether to grant your application the requested access. At this stage, SolveXia displays a consent form that shows the name of your application.
+The user can then authorise access to their SolveXia resources requested by your application or refuse the request.
+
 ![authorize](authorize.png)
 
-You will be redirected back to the supplied (or configured) redirect uri with authorization code in query string
-https://returnurl.com?code=834t0jgf03ht3h5tp03hhw3ph432trtj349t439jt034jh35ha34tj34hth34t034t3
+If user authorises access you will be redirected back to the supplied (or configured) redirect uri with authorization code in the query string.
```

Comment:
I think this makes more sense: "If the user authorizes access, you will be redirected back to the supplied (or configured) redirect uri with the authorization code in the query string."

Reply:
Thanks, fixed.